### PR TITLE
add O'Neill's formula for curvature

### PR DESCRIPTION
**Pull request by nguigs**
Add O'Neill's formula to compute the curvature of Riemannian submersions, and application in Kendall shape space.

### CODE REVIEW

File: geomstats/geometry/fiber_bundle.py [hunk 1]
```diff
@@ -33,8 +39,10 @@ class FiberBundle(Manifold):
         group. Either dim, base or group must be given as input.
     """
 
-    def __init__(self, total_space, base=None, group=None, group_action=None,
-                 dim=None, **kwargs):
+    def __init__(
```
Comment (ninamiolane) [inline geomstats/geometry/fiber_bundle.py:42]: Yes, we need to decide if we use typing everywhere! @MariusGuerard was also thinking about this.

File: tests/test_quotient_metric.py [hunk 2]
```diff
@@ -153,3 +153,14 @@ def test_squared_dist(self):
             points[1], points[0], tol=1e-10)
         expected = self.base_metric.squared_dist(points[1], points[0])
         self.assertAllClose(result, expected, atol=1e-5)
+
+    def test_fundamental_tensor(self):
```
Comment (ninamiolane) [inline tests/test_quotient_metric.py:157]: NIT: test_fundamental_a_tensor

**Review by ninamiolane — APPROVED** [review]
Very nice! Why the decrease in code coverage?

Also, we should decide if we want to add typing in the library. I think @MariusGuerard and @tristancabel could be pro-typing.

**Comment by nguigs** [comment]
I am surprised by the codecov report too, I used sourcegraph but didn't find the missing hits...

**Pull request merged**